packages/perf/perftool-testsuite/runtest.sh

```diff
@@ -222,8 +222,6 @@ rlJournalStart
 			which debuginfo-install || rlRun "$YUM -y install yum-utils dnf-utils" 0 "Installing {yum,dnf}-utils (it has not been present)"
 			which debuginfo-install # now it should be installed, but what if it fails...
 			if [ $? -eq 0 ]; then
-				rlRun "debuginfo-install -y $KERNEL_PKG_NAME" 0 "Installing debuginfo for $KERNEL_PKG_NAME via debuginfo-install (it has not been present)"
```

Comment:
I removed this since it fails to install using the rt debuginfo package using debuginfo-install, the second statement always works in both cases.